README.md

```diff
@@ -133,6 +134,25 @@ $ docker pull xcoo/cruler:${TAG}
 $ docker run --rm -v /path/to/validator/project:/cruler -it xcoo/cruler:${TAG}
 ```
 
+### Use as a library
+
+You can use Cruler as a library.
+
+```clojure
+(ns sample
+  (:require [cruler.core :as cc]))
+
+(defn run [_]
+  (let [[config-file-path config] (cc/setup-config "dev-resources/sample-validator" "cruler.edn")]
+    (println config-file-path)
+
+    ;; If you want to validate a single file, you can use "run-validators-single-file" method.
+    (println (cc/run-validators-single-file (:validators config) "dev-resources/sample-validator" "description/test.txt"))
```

Comment:
The output is 
```clojure
({:type :pass,
  :validator :sample-validator.sort/sort,
  :message "Sort error: not ascending order at line\n",
  :errors ()}
 {:type :pass,
  :validator :cruler.validators/start-of-file,
  :message "The file must not start with whitespaces\n",
  :errors ()}
 {:type :pass,
  :validator :sample-validator.csv-blank/csv-blank,
  :message "Blank column is found at line\n",
  :errors ()}
 {:type :pass,
  :validator :cruler.validators/blank-line,
  :message "Blank line is found at line\n",
  :errors ()}
 {:type :pass,
  :validator :cruler.validators/trailing-whitespace,
  :message "Trailing whitespaces must be removed\n",
  :errors ()}
 {:type :fail,
  :validator :sample-validator.spec/approval,
  :message
  "\ndev-resources/sample-validator/description/test.txt\n  error: Should be a map\n    [\"Abilify\"]\n  line: unknown\n\n\ndev-resources/sample-validator/description/test.txt\n  error: Should be a map\n    [\"Bacitracin\"]\n  line: unknown\n\n\ndev-resources/sample-validator/description/test.txt\n  error: Should be a map\n    [\"Cabergoline\"]\n  line: unknown\n\n\ndev-resources/sample-validator/description/test.txt\n  error: Should be a map\n    [\"Dabigatran\"]\n  line: unknown\n",
  :errors ()}
 {:type :pass,
  :validator :sample-validator.reference/approval<->drug,
  :message "Unknown drug ids:\n",
  :errors ()}
 {:type :pass,
  :validator :cruler.validators/end-of-file,
  :message "The file must end with a single newline.\n",
  :errors ()}
 {:type :pass,
  :validator :sample-validator.duplication/duplication,
  :message "Duplicated line is found at line\n",
  :errors ()})
```

`dev-resources/sample-validator/description/test.txt` looks to be validated by `sample-validator.spec/approval`. But in `cruler.edn`,  `description/test.txt` is not defined to be validated by `spec/approval`. Please check it.

Reply:
I fixed it in https://github.com/xcoo/cruler/pull/6/commits/e52301cf4ffa7eb3ee90202b10a1c5310a481251